app/infrastructure/mongodb/StowawayUser.js

```diff
@@ -0,0 +1,58 @@
+//@ts-check
+const User = require('../../domain/user/User');
+
+/**
+ * @typedef {import('../../domain/user/types').User} UserType
+ * @typedef {import('../../domain/user/types').Playlist} Playlist
+ * @typedef {import('./types').UserDocument} UserDocument
+ */
+exports.StowawayUser = class StowawayUser extends User {
+  #userDocument;
```

Comment:
Quick fix potentiel pour résoudre l'erreur TS `Private identifiers are only available when targeting ECMAScript 2015 and higher.`: choisir une `target` plus récente dans `tsconfig.json`.